bees/facebookbee/facebookbee.go

```diff
@@ -76,17 +78,169 @@ func (mod *FacebookBee) Run(eventChan chan bees.Event) {
 	}
 }
 
+func (mod *FacebookBee) handlePermanentPageToken() {
+	if mod.pageAccessToken != "" {
+		// return
+	}
+
+	mod.Logf("Attempting to fetch long lived user access token")
+
+	longToken := mod.fetchLongLivedUserAccessToken()
+
+	if longToken == "" {
+		mod.LogErrorf("No long lived user access token!")
+		return
+	}
+
+	// mod.Logf("Long lived user access token: \"%s\"", longToken)
+	accountID := mod.fetchAccountID(longToken)
+
+	if accountID == "" {
+		mod.LogErrorf("No account id!")
+		return
+	}
+
+	pageToken := mod.fetchPermanentPageAccessToken(accountID, longToken)
+	mod.Logf("pageToken: \"%s\"", pageToken)
+}
+
+func (mod *FacebookBee) fetchLongLivedUserAccessToken() string {
```

Comment:
I would recommend to additionally return any potential errors from this function (instead of just an empty string), so we can cleanly report/log them - at least eventually.

Reply:
Would be returning an error string sufficient?

Reply:
I'll create a 2nd error result.